docker/Dockerfile-dev

```diff
@@ -12,11 +22,12 @@ RUN microdnf update -y && microdnf install -y \
     python${PYTHON_VERSION}-pip python${PYTHON_VERSION}-devel wget && \
     microdnf clean all
 
+# hadolint ignore=SC1091
```

Comment:
If you explained the previous ones, explaining this might also make sense, but it's far from critical (because even the warning itself is a bit weird to me).

Reply:
see previous comments. Added.